specify/specify7
# feat(FormParse): allow "false" as default value for Boolean

Fixes #4581

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

1. Add a checkbox to the form with `default="true"`.
2. Open the form
3. Without touching the checkbox, save the resource to the database
4. Run a Query Builder query on the resource and make sure that in the query results, the checkbox field has true value
5. Do the above steps with `default="false"` and make sure that query results has false value
6. Do the above steps with no `default` attribute and make sure that query results cell for this value is empty
